Data_Structures_Questions.md

```diff
@@ -3,18 +3,21 @@ Answer the following questions for each of the data structures you implemented a
 ## Queue
 
 1. What is the runtime complexity of `enqueue`?
+   O(1)
 
 2. What is the runtime complexity of `dequeue`?
+   O(1)
 
-3. What is the runtime complexity of `len`?
+3) What is the runtime complexity of `len`?
+   O(1), length is stored as a property, and returned by the len method, we just need to get it no need for iteration thus O(1)
```

Comment:
awesome. yep, queues are cheap to implement for these methods. in fact, they're pretty cheap in general! the worst case complexities queues have are O(n) for both space and time (when performing accesses and searches).